.github/scripts/continuous-integration

```diff
@@ -24,8 +24,7 @@ if ($event === '--pull-request')
 {
 	$files = shell_exec('git diff --name-only --diff-filter=ACMRTUXB origin/develop..HEAD');
 }
-else
-{
+else {
```

Comment:
Please revert the changes made to this file. I think the fix is a bug in php-cs-fixer.

Reply:
ok, but it will fail tests

Reply:
Yes, that's the compromise for now.